src/components/MessageAttachments.vue

```diff
@@ -20,37 +20,52 @@
   -->
 
 <template>
-	<div class="mail-message-attachments">
-		<div class="attachments">
-			<MessageAttachment
-				v-for="attachment in attachments"
-				:id="attachment.id"
-				:key="attachment.id"
-				:file-name="attachment.fileName"
-				:size="attachment.size"
-				:url="attachment.downloadUrl"
-				:is-image="attachment.isImage"
-				:is-calendar-event="attachment.isCalendarEvent"
-				:mime="attachment.mime"
-				:mime-url="attachment.mimeUrl"
-				@click="showViewer(attachment)" />
-			<AttachmentImageViewer v-if="attachmentImageURL && showPreview"
-				:url="attachmentImageURL"
-				@close="showPreview = false" />
+	<div v-if="attachments.length > 0" class="mail-message-attachments" :class="hasNextLine ? 'has-next-line' : ''">
+		<div class="mail-message-attachments--wrapper" :class="(hasNextLine === true && isToggled === true) ? 'hide' : ''">
+			<div class="attachments">
+				<MessageAttachment
+					v-for="attachment in attachments"
+					:id="attachment.id"
+					ref="attachments"
+					:key="attachment.id"
+					:file-name="attachment.fileName"
+					:size="attachment.size"
+					:url="attachment.downloadUrl"
+					:is-image="attachment.isImage"
+					:is-calendar-event="attachment.isCalendarEvent"
+					:mime="attachment.mime"
+					:mime-url="attachment.mimeUrl"
+					@click="showViewer(attachment)" />
+				<AttachmentImageViewer v-if="attachmentImageURL && showPreview"
+					:url="attachmentImageURL"
+					@close="showPreview = false" />
+			</div>
+		</div>
+		<div v-if="hasNextLine"
+			class="show-more-attachments"
+			@click="isToggled = !isToggled">
+			<ChevronDown v-if="isToggled" :size="24" />
+			<ChevronUp v-if="!isToggled" :size="24" />
+			<span v-if="isToggled">
+				{{ n('mail', 'View more {count} attachment', 'View more {count} attachments', (attachments.length - visible), { count: attachments.length - visible }) }}
```

Comment:
```suggestion
				{{ n('mail', 'View {count} more attachment', 'View {count} more attachments', (attachments.length - visible), { count: attachments.length - visible }) }}
```

Or simpler: *View all attachments*

Reply:
Fixed, thanks!